apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationDetails.tsx

```diff
@@ -54,12 +54,13 @@ const ApplicationDetails = ({ onClose, entity }: Props) => {
   }, [entities, entity.name]);
 
   const handleUseEntity = useCallback(() => {
+    const queryParamId = searchParams.get(
+      MarketplaceQueryParams.fromConversation,
+    );
```

Comment:
it should be out of `useCallback` and memoized because can't be changed

Reply:
I think that's don't make sense to create a separate variable. We use this callback once. It not heavy at all to memoize it, we just take a parameter from the URI